load_data.py

```diff
@@ -0,0 +1,21 @@
+import yfinance as yf
+import pandas as pd
+import sqlite3
+import sqlalchemy
+
+def save_df_to_db(
```

Comment:
looks like you are missing the db setup/initialization. can you try calling this save_df_to_db to see i you are able to save data to sqlite db.